regexes/device/mobiles.yml

```diff
@@ -81,9 +81,15 @@ HTC:
 
     - regex: 'NexusHD2' # custom rom for hd2
       model: 'HD2'
-    - regex: 'HTC[ _\-]((?:Flyer|P715a).*) Build'
+    - regex: 'HTC[ _\-]P715a'
       device: 'tablet'
-      model: '$1'
+      model: 'P715a'
+    - regex: 'HTC[ _\-]Flyer Build'
+      device: 'tablet'
+      model: 'Flyer'
+    - regex: 'HTC[ _\-]Flyer[ _\-]([\w]{1,5})'
+      device: 'tablet'
+      model: 'Flyer $1'
     - regex: 'HTC[ _\-]One[ _\-]max[);/ ]'
```

Comment:
thought phablets are 5.5–6.99 inches, also for exemple here http://www.gsmarena.com/htc_flyer-3829.php it's marked as tablet

let me know what's piwik approach when something is phablet and when something is tablet :)